go.mod

```diff
@@ -135,7 +134,7 @@ replace github.com/gomodule/redigo => github.com/gomodule/redigo v1.8.2
 exclude google.golang.org/grpc/examples v0.0.0-20200728065043-dfc0c05b2da9
 
 // Used for /-/reload
-replace github.com/weaveworks/common => github.com/rfratto/weaveworks-common v0.0.0-20210326192855-c95210d58ba7
+replace github.com/weaveworks/common => github.com/rlankfo/weaveworks-common v0.0.0-20211028201502-f0486066ff0a
```

Comment:
I wish we didn't have to bounce around which repo we're using all the time :) Can we see if the changes I made to weaveworks/common can be moved to dskit + support the changes you made too?

(This might have to be a PR all on its own, honestly)

Reply:
no additional changes were made here aside from rebasing the `custom-registerer` branch from your fork on latest upstream changes. I agree bouncing around isn't ideal. Do you have an open PR to get the changes merged upstream? I'd prefer not to block this PR on figuring this out if we can avoid it.

Reply:
Yeah, there's these two PRs:

* https://github.com/weaveworks/common/pull/228
* https://github.com/weaveworks/common/pull/230

but I think since all movement is going to dskit it might be more worthwhile replaying the changes there instead with a PR.

Reply:
@rfratto I see that weaveworks/common#228 was merged. This PR doesn't attempt to pull in the weaveworks/common#230 but it seems like there's some movement there. For now looks like we can go back to weaveworks/common instead of this fork